pkgs/applications/audio/musescore/darwin.nix

```diff
@@ -1,9 +1,9 @@
 { stdenv, lib, fetchurl, undmg }:
 
 let
-  versionComponents = [ "4" "0" "1" ];
+  versionComponents = [ "4" "1" "0" ];
```

Comment:
Why is this not derived from version?

Reply:
Curious to know as well.

Reply:
> Why is this not derived from version?

Good question! I wasn't the one who wrote these expressions in the 1st place. Maybe the darwin expression is not needed anymore... I'll try to test this via ofborg a bit.